client/ayon_houdini/plugins/publish/collect_cache_farm.py

```diff
@@ -16,9 +16,11 @@ class CollectFarmCacheFamily(plugin.HoudiniInstancePlugin):
 
     def process(self, instance):
         creator_attribute = instance.data["creator_attributes"]
-        farm_enabled = creator_attribute["farm"]
-        instance.data["farm"] = farm_enabled
-        if not farm_enabled:
+        farm_enabled = creator_attribute.get("farm")
```

Comment:
Shouldn't this be `render_target` now?
```suggestion
        render_target = creator_attribute.get("render_target")
        if render_target != "farm":
            self.log.debug("Caching on farm is disabled. "
                   "Skipping farm collecting.")
            return

        instance.data["farm"] = True
```

Reply:
Forgot about adding a comment about backward/other current cache instances compatibility. 
other instances don't have `render_target` they only have `farm` creator attribute.

Reply:
Let me try, I hope it doesn't need further changes.

Reply:
I agree with @BigRoy that ditching `farm` BoolDef to avoid the confusion on the codes. It shouldn't take too long to make these changes.

Reply:
Thank you. 
https://github.com/ynput/ayon-houdini/pull/180/commits/1124f995898efcc10741a44e00ce24f278093556